spartan/aztec-network/templates/boot-node.yaml

```diff
@@ -234,16 +233,24 @@ metadata:
   labels:
     {{- include "aztec-network.labels" . | nindent 4 }}
 spec:
+  # If this is a public network, we want to expose the boot node as a LoadBalancer
+  {{- if .Values.network.public }}
+  type: LoadBalancer
```

Comment:
we had this in the past, right?

Reply:
Sort of. We had a load balancer, but I removed it for *public* networks because it didn't work as "external" when mixing tcp/udp. 

So this incarnation *always* creates a service (public/private), but when it is a public network, we don't try to allow it to be used as something you can do p2p through.